### PR TITLE
Update Oxidized.md

### PR DESCRIPTION
**Pull request by achrstl**
Hello,
I think this could be really useful, i searched for a long time before finding that command would work in a bare repo.
It allow to remove deleted devices that we don't have anymore.

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by murrant** [comment]
Hmm, I think you need to add a little more context here. To explain why people would run the command you listed.

We don't remove retired devices from our git repo because it is a tiny amount of storage and it is not visible in the ui or anything.

If you can add some more context, I'll be happy to merge.

**Comment by achrstl** [comment]
I use the oxidized repo to search for config part.
By example if i need to quickly look where the IP 1.2.3.4 is configured: `git grep 1.2.3.4`
The output, will return me which device is configured with this IP.
Now, if an old device had this IP, i will have two results.
In my case, i have some automation script that need to match with the current network state.

Thats why i think it could be relevant to add this command.

File: doc/Extensions/Oxidized.md [hunk 1]
```diff
@@ -252,3 +252,9 @@ file content:
 ```
 git cat-file -p <object id>
 ```
+
+# Remove disabled/removed device
```
Comment (PipoCanaja) [inline doc/Extensions/Oxidized.md]: As suggested by murrant, you could add something like : 
```
If you want to purge saved config of a device that is not in LibreNMS anymore, you can run the following command: 
```

**Comment by PipoCanaja** [comment]
Hi @achrstl 
Looks good ! Thanx for your contribution.

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-59-release-changelog-december-2019/10558/1

**Pull request merged**